docs/GettingStarted.md

```diff
@@ -513,6 +517,32 @@ The `sucker_punch` integration traces all scheduled jobs:
     pin = Datadog::Pin.get_from(::SuckerPunch)
     pin.service = 'deploy-queues'
 
+## Frontend web servers
+
+### Request Queue
+
+It's possible to compute the Request Queue time spent in your frontend web server or load balancer before
+requests reach your Ruby application. This functionality is **experimental** and is only available if you're
+using the Rack integration. To activate the feature, you must add a ``X-Request-Start`` header from your
+web server (i.e. nginx). The following is an nginx configuration example:
+
+    # /etc/nginx/conf.d/ruby_service.conf
+    server {
+        listen 8080;
+
+        location / {
+          proxy_set_header X-Request-Start "t=${msec}";
+          proxy_pass http://web:3000;
+        }
+    }
+
+Then you need to configure your Rack integration as follows:
+
+    Datadog.configure do |c|
+      # web_service_name defaults to `web-server` and is different from Rack `service_name`
+      c.use :rack, request_queuing: true, web_service_name: nginx
```

Comment:
Nit: `request_queuing` is a specific mechanism of the front web server, while here we want to more generally integrate with front web servers. Maybe we could have an option key more generic? Examples: `instrument_web|http_server`, `behind_web_server`, `http_server_enabled`, ...
I guess it also depends on what we will want to do once stable (enabled by default?).